doc/gazebo_simulation/gazebo_simulation.rst

```diff
@@ -0,0 +1,351 @@
+Just copy the following snippet and overwrite :code:`panda_moveit_config/config/ros_controllers.yaml` with it:
+
+.. code-block:: xml
+
+    # MoveIt-specific simulation settings
+    moveit_sim_hw_interface:
+        joint_model_group: controllers_initial_group_
+        joint_model_group_pose: controllers_initial_pose_
+    # Settings for ros_control control loop
+    generic_hw_control_loop:
+        loop_hz: 300
+        cycle_time_error_threshold: 0.01
+    # Settings for ros_control hardware interface
+    hardware_interface:
+        joints:
+            - panda_joint1
+            - panda_joint2
+            - panda_joint3
+            - panda_joint4
+            - panda_joint5
+            - panda_joint6
+            - panda_joint7
+            - panda_finger_joint1
+        sim_control_mode: 1  # 0: position, 1: velocity
+    # Publish all joint states
+    # Creates the /joint_states topic necessary in ROS
+    joint_state_controller:
+        type: joint_state_controller/JointStateController
+        publish_rate: 50
+    panda_arm_controller:
+        type: effort_controllers/JointTrajectoryController
+        joints:
+            - panda_joint1
+            - panda_joint2
+            - panda_joint3
+            - panda_joint4
+            - panda_joint5
+            - panda_joint6
+            - panda_joint7
+        gains:
+            panda_joint1: { p: 12000, d: 50, i: 0.0, i_clamp: 10000 }
+            panda_joint2: { p: 30000, d: 100, i: 0.02, i_clamp: 10000 }
+            panda_joint3: { p: 18000, d: 50, i: 0.01, i_clamp: 1 }
+            panda_joint4: { p: 18000, d: 70, i: 0.01, i_clamp: 10000 }
+            panda_joint5: { p: 12000, d: 70, i: 0.01, i_clamp: 1 }
+            panda_joint6: { p: 7000, d: 50, i: 0.01, i_clamp: 1 }
+            panda_joint7: { p: 2000, d: 20, i: 0.0, i_clamp: 1 }
+
+        constraints:
+            goal_time: 2.0
+        state_publish_rate: 25
+
+    panda_hand_controller:
+        type: effort_controllers/JointTrajectoryController
+        joints:
+            - panda_finger_joint1
+            - panda_finger_joint2
+
+        gains:
+            panda_finger_joint1: { p: 5, d: 3.0, i: 0, i_clamp: 1 }
+            panda_finger_joint2: { p: 5, d: 1.0, i: 0, i_clamp: 1 }
+
+        state_publish_rate: 25
+
+    controller_list:
+        - name: panda_arm_controller
+            action_ns: follow_joint_trajectory
+            type: FollowJointTrajectory
+            default: true
+            joints:
+            - panda_joint1
+            - panda_joint2
+            - panda_joint3
+            - panda_joint4
+            - panda_joint5
+            - panda_joint6
+            - panda_joint7
+        - name: panda_hand_controller
+            action_ns: follow_joint_trajectory
+            type: FollowJointTrajectory
+            default: true
+            joints:
+            - panda_finger_joint1
+            - panda_finger_joint2
+
```

Comment:
_action_ns_, _type_, _default_,  _joints_  should have the same indentation of _name_ both in _panda_arm_controller_ and _panda_hand_controller_   otherwise by just copy and pasting users will get the following error:

> ros_controllers.yaml contains invalid YAML:
mapping values are not allowed here
  in "<string>", line 63, column 18:
            action_ns: follow_joint_trajectory
                     ^
XML is <rosparam command="load" file="$(find panda_moveit_config)/config/ros_controllers.yaml"/>
The traceback for the exception was written to the log file

Reply:
Copy-pasting stuff between implicitly indented text editors (with their own rules, 2? 4? Or a tab, damn) is a long-lasting pain-in-the-neck for me :man_facepalming: Thanks for pointing it out :+1: